### PR TITLE
CapricornConsultingGmbH.Vigoon version 2.4.2261

### PR DESCRIPTION
**Pull request by cctimoerdbruegger**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41979)

### CODE REVIEW

File: manifests/c/CapricornConsultingGmbH/Vigoon/2.4.2261/CapricornConsultingGmbH.Vigoon.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: 2.4.2261
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/2.4.2261/CapricornConsultingGmbH.Vigoon.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/CapricornConsultingGmbH/Vigoon/2.4.2261/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: 2.4.2261
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/2.4.2261/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/CapricornConsultingGmbH/Vigoon/2.4.2261/CapricornConsultingGmbH.Vigoon.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: 2.4.2261
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/2.4.2261/CapricornConsultingGmbH.Vigoon.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This must match Control Panel:
image

Otherwise your application will be downloaded an unlimited amount of times whenever someone does `winget upgrade CapricornConsultingGmbH.Vigoon` or `winget upgrade --all`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Version mismatch.

**Comment by cctimoerdbruegger** [comment]
Ah, no i understand. You mean the version of the .msi file. Okay, i uploaded a new pull request with correct version and will close this one. thanks

**Pull request closed without merging**